src/GitVersion.Core.Tests/VersionCalculation/Strategies/VersionInBranchNameBaseVersionStrategyTests.cs

```diff
@@ -80,6 +82,7 @@ public void CanTakeVersionFromNameOfConfiguredReleaseBranch(string branchName, s
         var repository = fixture.Repository.ToGitRepository();
 
         var configuration = GitFlowConfigurationBuilder.New
+            .WithLabelPrefix("([vV]|lts-)?")
```

Comment:
Do we need to document this change being required, perhaps?

Reply:
What is requirement behind it to support lts before the number will be present? For sure if you define support/2.0.0-lts it works per default if you define support/lts-2.0.0 it's not working per default and a change in the configuration needs to be done (either changing `label-prefix` or `version-in-branch-name`). I think this line is not necessary anymore because I have changed the test case.

Reply:
I have added a new test DoesNotTakeVersionFromNameWhenItHasBeenAccidentalSpecifiedInBranch